README.md

```diff
@@ -12,115 +12,131 @@ npx knex migrate:latest
 npx knex seed:run
 cp .env.example .env
 ```
+
 To run in development:
+
 ```sh
 npm run dev
 ```
 
 To run in production:
+
 ```sh
 npm start
 ```
+
 ## User Stories
 
 ### MVP
 
-* As a developer who is new to the project, I want to see an informative README, so that I can get started with the code right away.
-* As a developer, I want the linter and unit tests to run automatically before I make new commits, so that I don't have to remember to run them. (Husky)
-* As a user, I want to join an event, so that I can keep track of the events I am participating in in Coject.
-* As a user, I want to be added to an event when I click the Join button.
+- As a developer who is new to the project, I want to see an informative README, so that I can get started with the code right away.
+- As a developer, I want the linter and unit tests to run automatically before I make new commits, so that I don't have to remember to run them. (Husky)
+- As a user, I want to join an event, so that I can keep track of the events I am participating in in Coject.
+- As a user, I want to be added to an event when I click the Join button.
 
 ## Stretch
 
-* As a user, once logged in I want to have tabs to navigate the App.
+- As a user, once logged in I want to have tabs to navigate the App.
 
 ---
 
 ## Views (Client Side)
-| name | purpose |
-| --- | --- |
-| Login | View for user to enter their login credentials |
-| Register | View for user to sign up for the App |
-| Home | Placeholder |
-| Create Project | Placeholder |
-| Events | Placeholder |
-
-## Reducers (Client Side)
 
-| name | purpose |
-  | --- | --- |
-  | auth | Store information regarding user logins, auth status and auth errors |
-  | Create User | Placeholder |
-  | Current Page | Placeholder |
-  | Set Events | Placeholder |
+| name           | purpose                                        |
+| -------------- | ---------------------------------------------- |
+| Login          | View for user to enter their login credentials |
+| Register       | View for user to sign up for the App           |
+| Home           | Placeholder                                    |
+| Create Project | Placeholder                                    |
+| Events         | Placeholder                                    |
```

Comment:
❔ These will probably go out-of-date very quickly as the app grows; instead maybe consider adding comments to the view classes, reducer functions, etc. to document their purpose (related:  #6 #4 #5 ).

Reply:
💯 Agree...... using docblocks for the relevant components is a great way to document their purpose